modules/polaris/src/main/ts/ephox/polaris/api/Regexes.ts

```diff
@@ -28,6 +28,7 @@
   - allow up to 15 character schemes including all valid characters from the spec https://url.spec.whatwg.org/#url-scheme-string (TINY-5074)
   - changed instances of 0-9 to be \d (TINY-5074)
   - reduced duplication (TINY-5074)
+  - include additional valid characters in the path segment (TINY-8069)
```

Comment:
```suggestion
  - included additional valid characters `!` and `:` in the path segment (TINY-8069)
```

Reply:
We included more than just `!` though, so that wouldn't be accurate. You're right it should be `included` though.

Reply:
Looking at the diff, the full list is: `*`, `!`, `;`, `:`, `', `@`, `^` and `$`.

Reply:
Reading the spec again though `^` shouldn't be there though (I copied it largely from what was in the query string so looks like I missed removing that).

Reply:
Fixed that invalid character and reworded this message.

Reply:
Right the changelog item for polaris confused me to think that it was only these characters and also the ticket information. But good to have a full list and spec reference here. Thanks.